AUTHORS

```diff
@@ -22,3 +22,4 @@ Philipp Horn <unicorn@wh2.tu-dresden.de>
 Sebastian Schrader <shreyder@wh2.tu-dresden.de>
 Till Smejkal <till.smejkal@mailbox.tu-dresden.de>
 Tim Kluge <timklge@wh2.tu-dresden.de>
+Stefan Haller <stefanhaller@wh2.tu-dresden.de>
```

Comment:
Please adhere to the alphabetical order (I'm sorry @strebbin, it appears you inherited some issues ;) )

Reply:
To be honest, I did expect to contribute more code to pycroft than such a little unfinished change. :smile: